ui/src/main/webapp/wizards/sources/actions.js

```diff
@@ -1,6 +1,7 @@
 import { post } from 'redux-fetch'
 
 import { getAllConfig } from '../../reducer'
+import { getDiscoveryConfigs } from './reducer'
```

Comment:
We were actually wanting to migrate to a different method of setting the root selector for these in the root reducer. I'll make that change real quick and it will clean up these imports nicely so that everything only ever imports selectors from their respective reducers instead of having to traverse to the root reducer.